### DISCUSSION TITLE
Provide page with description of the mostly used plugins

### DISCUSSION THREAD

**Discussion by hendrikebbers** [support-and-care/maven-support-and-care#2] [Ideas for Funding]
Since Maven is based on a plugin system there are a lot of different and open source maven plugins. Some of them are very common and used in every or mostly every Maven based Java project. We should identify that 10-20 plugins and create a page that names and describes those plugins. Such a page can be used to show information about the latest version and the security standards (scores) of those plugins, too.

**Comment by taijidude** [comment]
I think that would be very helpful. I often have no clue in how to find the right plugin for the job and then default to bash or powershell scripts.

**Comment by ascheman** [comment]
This discussion has been converted to Epic #58 for tracking.

Closing this discussion.
